crates/dojo-world/src/contracts/world_test.rs

```diff
@@ -35,7 +35,7 @@ pub async fn deploy_world(
     sequencer: &TestSequencer,
     path: Utf8PathBuf,
 ) -> (FieldElement, FieldElement) {
-    let manifest = Manifest::load_from_path(path.join("manifest.json")).unwrap();
+    let manifest = World::load_from_path(path.join("manifest.json")).unwrap();
```

Comment:
Shouldn't be a directory now when calling this `load_from_path`?

Reply:
yeah tests haven't been updated yet, still WIP